tasks/src/tabular-regression/about.md

```diff
@@ -3,7 +3,18 @@
 Tabular regression is the task of predicting a numerical value given a set of attributes/features. *Tabular* meaning that data is stored in a table (like an excel sheet), and each sample is contained in its own row. The features used to predict our target can be both numerical and categorical. However, including categorical features often requires additional preprocessing/feature engineering (a few models do accept categorical features directly, like [CatBoost](https://catboost.ai/)). An example of tabular regression would be predicting the weight of a fish given its' species and length.
 
 ## Use Cases
+Tabular data is one of the most prevalent data out there, and many raw-datasets once we extract features from them can be arrange in tabular form. Below you can find some of the popular tabular use-cases:-
 
+- **Preditcting a continuous target value**: 
+  When the data has been shared to us as a table, say in a spreadsheet in this case we can use the set of attributes/features to predict our target. Some popular examples like, predicting Taxi Fare based on distance traveled and time taken, predicting the Price of House based on number of rooms, square ft. area,  etc.
+
+- **Time Series Forecasting**:
+  Almost all time series forecasting problems can be solved as a tabular regression problem. Popular examples could be predicting the temperature next day, forecasting the Sales of a store across time.
+
+- **Filling Missing Values**:
+  This would be a little different from above examples however, suppose you have a table of data with some values that missing across multiple columns and rows. Here the challenge we address is of filling in those missing values.  
+
+## Metrics Used
```

Comment:
You can use this template and make your changes inside it for convenience, for others to contribute easier and also for us to review easier: https://github.com/huggingface/hub-docs/blob/main/tasks/src/placeholder/about.md

Reply:
@merveenoyan Alright I will update things here. One question was for use cases primarily I have seen are either predicting a continuous variable or doing imputation. 
1. Are there any other use-cases you can suggest?
2. For predicting continuous values would possible task variants be predicting trip duration, sales, housing price, temperature etc. or like predicting single values and predicting multiple values?

Reply:
You don't necessarily need to add more use cases but a bit more explanation for each of them should be fine 🙂 For latter, I think it's good to keep it simple but I will leave it up to you! Feel free to add multiple target prediction if you think it's common.